pkg/infra/config_test.go

```diff
@@ -9,20 +10,25 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("Config", func() {
+const DUMMY_TLS = "dummy.pem"
+const PEM_BUF = "a"
 
-	Context("config", func() {
-		It("successful load", func() {
-			var configText = `
+type Values struct {
+	TlsFile string
+}
```

Comment:
not a big deal but this could be removed if you use anonymous struct at line 86:
```go
generateConfigFileForTest(f.Name(), struct{ TlsFile string }{tlsFile.Name()})
```